addons/main/script_mod.hpp

```diff
@@ -10,7 +10,7 @@
 #define VERSION_AR  MAJOR,MINOR,PATCHLVL,BUILD
 
 // MINIMAL required version for the Mod. Components can specify others..
-#define REQUIRED_VERSION 2.14
+#define REQUIRED_VERSION 2.16
```

Comment:
```suggestion
#define REQUIRED_VERSION 2.14
```
Without `focusOn`, this can remain 2.14.